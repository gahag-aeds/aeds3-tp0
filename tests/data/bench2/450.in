450 225
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 49 95 71 5 41 16 5 19 2 97 68 0 83 35 2 59 74 88 4 24 16 36 2 52 98 62 94 33 39 25 51 89 20 22 46 62 90 51 33 92 
Add 1 150
Sum 102 251
Add 45 194
Max 284 433
Sub 90 239
Max 299 448
Add 143 292
Sum 264 413
Add 139 288
Max 200 349
Sub 140 289
Min 256 405
Sub 236 385
Max 194 343
Sub 8 157
Sum 87 236
Sub 56 205
Max 254 403
Sub 120 269
Max 95 244
Sub 132 281
Sum 72 221
Sub 236 385
Max 152 301
Sub 246 395
Sum 282 431
Sub 166 315
Sum 237 386
Add 33 182
Sum 99 248
Add 181 330
Min 34 183
Add 253 402
Max 278 427
Add 69 218
Sum 42 191
Sub 28 177
Max 212 361
Add 57 206
Max 167 316
Add 163 312
Min 196 345
Sub 122 271
Sum 150 299
Add 89 238
Min 181 330
Sub 236 385
Min 196 345
Sub 186 335
Max 56 205
Sub 42 191
Max 17 166
Add 179 328
Max 278 427
Add 169 318
Min 124 273
Add 259 408
Max 86 235
Sub 112 261
Max 44 193
Add 237 386
Sum 45 194
Add 129 278
Sum 189 338
Sub 74 223
Max 197 346
Sub 282 431
Max 101 250
Sub 160 309
Max 38 187
Add 267 416
Max 74 223
Add 233 382
Max 140 289
Sub 224 373
Min 73 222
Sub 20 169
Sum 159 308
Sub 268 417
Max 206 355
Sub 266 415
Sum 9 158
Sub 222 371
Sum 144 293
Sub 38 187
Min 142 291
Sub 20 169
Min 49 198
Sub 280 429
Sum 183 332
Sub 92 241
Sum 216 365
Add 279 428
Max 272 421
Sub 156 305
Min 52 201
Sub 220 369
Max 137 286
Add 153 302
Sum 132 281
Sub 226 375
Sum 171 320
Add 205 354
Max 158 307
Add 11 160
Max 128 277
Sub 282 431
Sum 30 179
Add 287 436
Sum 249 398
Add 287 436
Sum 252 401
Add 9 158
Sum 261 410
Sub 148 297
Max 47 196
Sub 154 303
Sum 219 368
Add 95 244
Min 133 282
Add 101 250
Min 238 387
Sub 100 249
Max 80 229
Add 261 410
Sum 255 404
Add 183 332
Min 232 381
Add 143 292
Max 35 184
Add 63 212
Max 68 217
Sub 258 407
Min 268 417
Add 277 426
Min 268 417
Add 95 244
Sum 258 407
Add 49 198
Min 133 282
Sub 258 407
Sum 36 185
Add 137 286
Sum 267 416
Sub 48 197
Max 284 433
Add 65 214
Sum 201 350
Sub 202 351
Sum 159 308
Sub 34 183
Sum 54 203
Sub 148 297
Max 185 334
Add 185 334
Min 160 309
Sub 140 289
Max 68 217
Add 91 240
Sum 282 431
Sub 154 303
Min 154 303
Sub 50 199
Sum 111 260
Add 121 270
Max 26 175
Sub 78 227
Sum 267 416
Sub 36 185
Max 179 328
Sub 152 301
Max 293 442
Sub 266 415
Sum 288 437
Add 259 408
Min 13 162
Add 23 172
Max 23 172
Add 213 362
Sum 276 425
Add 233 382
Min 298 447
Add 29 178
Sum 132 281
Add 183 332
Min 214 363
Add 291 440
Max 74 223
Add 281 430
Min 82 231
Add 55 204
Max 134 283
Add 287 436
Sum 156 305
Add 297 446
Max 107 256
Sub 234 383
Sum 126 275
Add 125 274
Max 269 418
Sub 4 153
Sum 276 425
Add 261 410
Max 269 418
Add 15 164
Max 272 421
Add 281 430
Min 37 186
Sub 294 443
Sum 246 395
Sub 12 161
Sum 279 428
Add 295 444
Sum 93 242
Sub 110 259
Min 229 378
Sub 6 155
Sum 153 302
Sub 2 151
Max 38 187
Sub 234 383
Min 109 258
Add 223 372
Max 272 421
Sub 16 165
