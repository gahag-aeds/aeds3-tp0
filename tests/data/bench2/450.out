7640
99
99
7581
100
0
98
7552
98
98
7154
97
7072
6948
7196
-2
99
7360
98
98
-2
7553
-3
-4
101
100
99
-3
98
100
7336
7733
99
97
99
99
96
-1
7653
98
6883
7271
-3
-4
7340
7001
98
-6
92
7130
7219
93
93
6732
6899
7026
7074
98
6796
-5
-6
94
7171
-4
98
99
-4
-3
7206
-3
7259
7154
99
7172
7567
7481
97
-4
99
7040
-6
7396
98
7050
97
100
7121
-5
97
7092
-3
7372
-4
98
-4
97
7523
97
7483
103
7696
104
104
-2
7652
7937
7588
-3
7517
96
-3
105
